packages/dds-components/src/components/Tabs/TabList.tsx

```diff
@@ -11,15 +11,21 @@ import {
 } from 'react';
 
 import { AddTabButton } from './AddTabButton';
+import { type TabProps } from './Tab';
 import { useTabsContext } from './Tabs.context';
 import styles from './Tabs.module.css';
 import { TabWidthContextProvider } from './TabWidthContext';
 import { useCombinedRef, useRoveFocus } from '../../hooks';
-import { cn } from '../../utils';
+import { cn, combineHandlers } from '../../utils';
 import { focusable } from '../helpers/styling/focus.module.css';
 import { scrollbar } from '../helpers/styling/utilStyles.module.css';
 
-export type TabListProps = HTMLAttributes<HTMLDivElement>;
+export type TabListProps = {
+  /**
+   * `<Tab>`-barn.
+   */
+  children?: Array<ReactElement<TabProps>> | ReactElement<TabProps>;
+} & Omit<HTMLAttributes<HTMLDivElement>, 'children'>;
```

Comment:
Tror ikke dette faktisk funker. Er en begrensning i TypeScript hvor du ikke kan beskrive hvilke React-elementer som skal kunne sendes inn. https://github.com/typescript-cheatsheets/react/issues/271 beskriver det fint.

Prøvde kjapt å se om jeg kunne sende andre React-elementer inn som barn (la f.eks. inn en `<div>`) i en story og ser ikke at jeg får noen typefeil på det. Tenker vi derfor ikke trenger denne endringen, og dermed ikke trenger å ha en major-changeset på det. 

```suggestion
export type TabListProps = HTMLAttributes<HTMLDivElement>;
```

Reply:
Grunnen til den endringen var å håndtere en typefeil; Vi vil få taket i `onClick`-en til `<Tab>` barn får å kunne slå den sammen med `onClick` fra kloningen. Hvis vi har generisk `children` får vi at typen til props er `unknown`:
![bilde](https://github.com/user-attachments/assets/e0dee4fe-4a48-41a5-8240-af786088b9f2)
Har du andre forslag?

Reply:
Sikkert enklest å sende inn TabProps til `isValidElement`:

<img width="519" alt="image" src="https://github.com/user-attachments/assets/c322d5c0-8a2d-4868-892c-c262543f1f62" />

Reply:
Godt poeng! Jeg tenkte for komplisert 😄 fikset.